### DISCUSSION TITLE
Feature Request: Add to Plex Watchlist via PlexAPI

### DISCUSSION THREAD

**Discussion by Tantumonium** [BenjaminOddou/alfred-plex#11] [Ideas]
When using "Plex Discover" and "Plex Search", it would be great to at least be able to Add specified items to my Watchlist!

(removal could also be useful, and an indication if the item is in the Watchlist already would probably be useful too)

**Comment by BenjaminOddou** [comment]
Hello @Tantumonium, I am working on the next version, it will include this feature ! stay tuned.

**Comment by BenjaminOddou** [comment]


**Reply by Tantumonium** [reply]
Adding Search or Discovery items to my Watchlist, works great! Thank you so much for your quick addition of this feature!
